.circleci/config.yml

```diff
@@ -317,7 +317,7 @@ commands:
             cd packages/functional-tests/tests
             TEST_FILES=$(circleci tests glob "./**/*.spec.ts")
             cd ..
-            echo $TEST_FILES | circleci tests run --command="xargs yarn playwright test --project=<< parameters.project >> $GREP" --verbose --split-by=timings
+            echo $TEST_FILES | circleci tests run --command="xargs yarn playwright test --project=<< parameters.project >> $GREP" --verbose --split-by=filesize
```

Comment:
The more I look, the less I understand.
https://circleci.com/docs/use-the-circleci-cli-to-split-tests/#playwright-tests
But it looks like we _are_ calling `store_test_results` on L342-343 below. Is the difference ours looks like a dir and the link above looks like a results.xml file? I almost need to print our config and the. CircleCI example and compare line for line. Or take a better look at the CI logs where it says it can’t find timing data and compare that to the number of tests it’s running to see if it had timing data for one shard but not the other 7.

Reply:
Yeah... there may actually be something wrong with `TEST_FILES=$(circleci tests glob "./**/*.spec.ts")`... investigating

Reply:
Bonus points if we echo that and pipe it to the artifacts dir. not sure if there is anything interesting there, or probably just a list of space/newline delimited file names.

Reply:
Specifically I think the reason we can't use '--split-by=timings' is because we are missing the 'files' attribute in our JUnit XML file ( see: https://circleci.com/docs/parallelism-faster-jobs/#junit-xml-reports)

Example:
<img width="568" alt="image" src="https://github.com/mozilla/fxa/assets/3422688/facc9f50-bc60-4929-bde0-7d9de35bf900">


Snippet from our files:
<img width="1117" alt="image" src="https://github.com/mozilla/fxa/assets/3422688/dac5b336-fd2b-4548-9d0b-ab5689634254">